akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -593,7 +593,7 @@ final class GraphInterpreter(
       while (eventsRemaining > 0 && queueTail != queueHead) {
         val connection = dequeue()
         eventsRemaining -= 1
-        chaseCounter = math.min(ChaseLimit, eventsRemaining)
+        chaseCounter = if (fuzzingMode) 0 else math.min(ChaseLimit, eventsRemaining)
```

Comment:
ok, my thought was that fuzzingMode not compile time constant and therefore the if would maybe add more overhead (hotspot might take care of it)

Would it be worth to move to a val outside the loop?

Not important